From: uazo <uazo@users.noreply.github.com>
Date: Mon, 19 Jan 2026 12:00:34 +0000
Subject: Disable device bound session

Disable DBSC due to potential privacy issues
see https://w3c.github.io/webappsec-dbsc/#privacy-considerations

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
 .../metrics/chrome_browser_main_extra_parts_metrics.cc      | 6 ++++++
 net/features.gni                                            | 2 +-
 2 files changed, 7 insertions(+), 1 deletion(-)

diff --git a/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc b/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
--- a/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
+++ b/chrome/browser/metrics/chrome_browser_main_extra_parts_metrics.cc
@@ -124,6 +124,10 @@
 #include "chrome/browser/signin/bound_session_credentials/unexportable_key_service_factory.h"
 #endif  // BUILDFLAG(IS_MAC)
 
+#if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
+#include "net/net_buildflags.h"
+#endif
+
 namespace {
 
 // The number of restarts to wait until removing the enable-benchmarking flag.
@@ -884,7 +888,9 @@ void RecordStartupMetrics() {
 #endif  // BUILDFLAG(IS_WIN)
 
 #if BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
+#if BUILDFLAG(ENABLE_DEVICE_BOUND_SESSIONS)
   crypto::MaybeMeasureTpmOperations(unexportable_keys::GetDefaultConfig());
+#endif
 #endif  // BUILDFLAG(IS_WIN) || BUILDFLAG(IS_MAC)
 
   // Record whether Chrome is the default browser or not.
diff --git a/net/features.gni b/net/features.gni
--- a/net/features.gni
+++ b/net/features.gni
@@ -57,7 +57,7 @@ declare_args() {
   chrome_root_store_only = is_win || is_mac || is_linux || is_chromeos
 
   # DBSC is only supported on Windows, Linux and Mac for now.
-  enable_device_bound_sessions = is_win || is_linux || is_mac
+  enable_device_bound_sessions = false
 
   # Bracketed URIs parsing is only available for debug builds.
   # TODO(crbug.com/365771838): Ensure tests are updated if needed if this
--
